wg-batch/annual-report-2024.md

```diff
@@ -2,32 +2,136 @@
 
 2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
 
+  Yes, JobSet has 1 active owner at the moment.
+
+### 2024 Highlights
+
+We will breakdown our highlights into Sub Projects, KEPs, talks, community adoption.
+
+#### Sub Projects
+
+##### Kueue
+
+Kueue has had 5 releases in 2024.
+
+- [Release 0.6](https://github.com/kubernetes-sigs/kueue/releases/tag/v0.6.0)
+
+- [Release 0.7](https://github.com/kubernetes-sigs/kueue/releases/tag/v0.7.0)
+
+- [Release 0.8](https://github.com/kubernetes-sigs/kueue/releases/tag/v0.8.0)
+
+- [Release 0.9](https://github.com/kubernetes-sigs/kueue/releases/tag/v0.9.0)
+
+- [Release 0.10](https://github.com/kubernetes-sigs/kueue/releases/tag/v0.10.0)
+
+In 2024, the kueue community would like to highlight are Topology aware scheduling, MultiKueue, Kueue Dashboard, KueueCtrl, Deployment/Statefulset integration for serving and Fair sharing.
+
+Topology aware scheduling facilitates scheduling of workloads that take in account data center topology. Workloads benefit from using interconnects that are physically close together.
+
+MultiKueue provides a way of dispatching batch workloads to worker clusters. Kueue provides multicluster dispatching for popular batch workloads such as Ray, Job, Kubeflow and JobSet. This feature went beta in 0.9.
+
+Kueue Dashboards has been a popular ask for Kueue. Users would like to have a visualization representation of queueing and we are happy to announce that a dashboard has been created for Kueue. This went into kueue in late 2024 and a big focus of 2025 will be to harden this for production.
+
+KueueCtrl provides a cli for creating kueue objects. The plugin is hosted in krew and is easily installed as a kueue plugin.
+
+Deployment/StatefulSet integration provides an avenue for the usage of Kueue for serving workloads. Serving leads to a need for sharing/preemption of model servers that may leverage accelerators. Kueue provides an integration with popular methods of deploying services (Deployment/StatefulSet).
+
+##### JobSet
+
+Jobset has had 4 release in 2024.
+
+- [Release 0.4](https://github.com/kubernetes-sigs/jobset/releases/tag/v0.4.0)
+
+- [Release 0.5](https://github.com/kubernetes-sigs/jobset/releases/tag/v0.5.0)
+
+- [Release 0.6](https://github.com/kubernetes-sigs/jobset/releases/tag/v0.6.0)
+
+- [Release 0.7](https://github.com/kubernetes-sigs/jobset/releases/tag/v0.7.0)
+
+A major achievement of JobSet has been the adoption of JobSet as a component for Kubeflow Training Operator V2.
+There has been a collaborative effort with the Kubeflow community and the batch community to implement the features needed for this integration.
+
+[Metaflow](https://github.com/Netflix/metaflow/pull/1804) has adopted the use of JobSet for distributed ML training.
+
+##### KJob
+
+[KJob](https://github.com/kubernetes-sigs/kjob?tab=readme-ov-file#kjob) has been started to provide a CLI friendly way for users to submit batch jobs.
```

Comment:
```suggestion
[KJob](https://github.com/kubernetes-sigs/kjob) has been started to provide a CLI friendly way for users to submit batch jobs.
```

Do we really need to add the Markdown tag?

Reply:
No I don't. Updated.